src/Test.FunctionalTests.Sarif/TestData/Partitioning/FilterByPredicate/Inputs/FilterByPredicateWithArtifacts.sarif

```diff
@@ -0,0 +1,113 @@
+{
+  "$schema": "https://schemastore.azurewebsites.net/schemas/json/sarif-2.1.0-rtm.4.json",
```

Comment:
>$schema [](start = 3, length = 7)

Despite this file's name (FilterByPredicateWithArtifacts), I will _not_ create new files when I implement filtering of logical locations, code flows, etc. I'll change the name of the file to FilterByPredicateWithRunLevelProperties and use it for all of them.

Actually maybe better is to get rid of the test file that has _not_ run-level properties, and give its name (FilterByPredicate) to this file.

Reply:
Done.

---
In reply to: [337815072](https://github.com/microsoft/sarif-sdk/pull/1708#discussion_r337815072) [](ancestors = 337815072)